test/TestHelper.h

```diff
@@ -97,6 +97,17 @@ namespace test
 	}																	\
 	while (0)
 
+struct ImportStateOptions
+{
+	ImportStateOptions():m_bHasBalance(false), m_bHasNonce(false), m_bHasCode(false), m_bHasStorage(false)	{}
```

Comment:
You do not need this constructor. Just use ImportStateOptions(bool _bSetAll = false) in the other constructor. This way _bSetAll will default to false if not set explicitly.